rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -716,13 +716,24 @@ impl SealevelMailboxIndexer {
         let hyperlane_message =
             HyperlaneMessage::read_from(&mut &dispatched_message_account.encoded_message[..])?;
 
-        let log_meta = self
-            .dispatch_message_log_meta(
-                U256::from(nonce),
-                &valid_message_storage_pda_pubkey,
-                &dispatched_message_account.slot,
-            )
-            .await?;
+        // let log_meta = self
+        //     .dispatch_message_log_meta(
+        //         U256::from(nonce),
+        //         &valid_message_storage_pda_pubkey,
+        //         &dispatched_message_account.slot,
+        //     )
+        //     .await?;
+
+        let log_meta = LogMeta {
+            address: self.program_id.to_bytes().into(),
+            block_number: dispatched_message_account.slot,
+            // TODO: get these when building out scraper support.
+            // It's inconvenient to get these :|
+            block_hash: H256::zero(),
+            transaction_id: H512::zero(),
+            transaction_index: 0,
+            log_index: U256::zero(),
```

Comment:
ditto here

Reply:
It was defaulted to zero before, so, should be good.